metadata.json

```diff
@@ -9,8 +9,9 @@
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
```

Comment:
Missing a comma on this line too, the file is invalid:

    Debug: ntp has an invalid and unparsable metadata.json file.  The parse error: 399: unexpected token at '{"name":"puppet/tea","version_requirement":"= 0.2.1"}